integreat_cms/static/src/js/events/event-query-pois.ts

```diff
@@ -42,6 +57,10 @@ const renderPoiData = (
         );
     document.getElementById("poi-query-result").classList.add("hidden");
     (document.getElementById("poi-query-input") as HTMLInputElement).value = "";
+    showContactFieldBox();
+    updateField("email", (document.getElementById("id_email_from_location") as HTMLInputElement).checked);
+    updateField("phone_number", (document.getElementById("id_phone_number_from_location") as HTMLInputElement).checked);
+    updateField("website", (document.getElementById("id_website_from_location") as HTMLInputElement).checked);
```

Comment:
I interpret the comment *checkbox by default ausgewählt, wenn Daten ausgefüllt, sonst unchecked* in the design in that way that the checkboxes should be checked by default if and only if their corresponding fields in the POI are set when creating the contact.

Also the fields taken from the model should be sorted below those that are set explicitly, though I'm not sure this is worth actually implementing and would argue that it might just add confusion.

Reply:
@nikolahoff 
These two behaviours are technically not simple 😿 and we think those may be rather confusing. Are they absolutely necessary?

![image](https://github.com/user-attachments/assets/306c94f0-1711-4885-8ba9-ca85bf7998b7)

Reply:
Checking whether the checkbox should be initialized as checked should be fairly simple though: We already have the information from the location, we copy it into the field after all. We can check if it is empty or not and check the box accordingly.

Reply:
@PeterNerlich You're right😁 Though, changing the order depend on if it is filled in the POI, is (not impossible but tough).

But anyway, both are confusing in my opinion 🤔

Reply:
@nikolahoff 
>But if it's too complicated why can skip it. But would it be ok to have the 2 Pflichtfelder on top always?
Yes, title and name stay on top :)

Reply:
![image](https://github.com/user-attachments/assets/5307ecd3-a8af-4aa5-a1d5-f24bed367c9a)


🤔 ?